src/ui/public/query_bar/components/query_bar.tsx

```diff
@@ -66,14 +66,25 @@ interface Query {
   language: string;
 }
 
+interface DateRange {
```

Comment:
This could probably go in the `timefilter` directory and be imported in the file I suggested above instead of declaring it in both places.

Reply:
The TimeRange from TimeHistory is not the same object as the DateRange defined in this file. `DateRange` describes the results returned to onSubmit. They could be a different form than `TimeRange`